packages/cozy-harvest-lib/src/components/DisconnectedAccountModal.jsx

```diff
@@ -69,9 +69,12 @@ const DisconnectedModal = ({ accounts, onClose, initialActiveTab }) => {
       ) : null}
       {activeTab === 1 ? (
         <>
-          {accounts.length ? (
+          {accounts.length && konnectorRef.current ? (
```

Comment:
Do we really want to display "No Contract" if the "konnectorRef" is not there? 

I think we should handle that higher in the component. Here we already pass konnector.current without any check https://github.com/cozy/cozy-libs/blob/1e4e4ccd72af2b33e7a9deb66efbe52006cf6ee7/packages/cozy-harvest-lib/src/components/DisconnectedAccountModal.jsx#L55 

What do you think about having: 
```
render(){
if(!konnectorRef.current){
return null // or <Loader />
}else{
...
}

Reply:
Right that's better